src/Microsoft.Identity.Client/ClientApplicationBase.WithBuilders.cs

```diff
@@ -25,14 +25,67 @@
 // 
 // ------------------------------------------------------------------------------
 
+using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.ApiConfig;
 using Microsoft.Identity.Client.AppConfig;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Exceptions;
+using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Client.PlatformsCommon.Factories;
+using Microsoft.Identity.Client.TelemetryCore;
+using Microsoft.Identity.Client.Utils;
 
 namespace Microsoft.Identity.Client
 {
     public abstract partial class ClientApplicationBase
     {
         internal ClientApplicationBase(ApplicationConfiguration config)
         {
+            ServiceBundle = Core.ServiceBundle.Create(config);
+
+            UserTokenCache = config.UserTokenCache;
+            if (UserTokenCache != null)
+            {
+                UserTokenCache.ClientId = ClientId;
+                UserTokenCache.ServiceBundle = ServiceBundle;
+            }
+
+            CreateRequestContext(Guid.Empty).Logger.Info(string.Format(CultureInfo.InvariantCulture,
```

Comment:
>Info [](start = 52, length = 4)

should this be only in the verbose setting? I don't have a clear meaning on this myself, just wondering why you considered

Reply:
This is the level it was at before.  I can change it to make it verbose, but I'm trying not to make behavioral changes whenever possible.

---
In reply to: [247210395](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/780#discussion_r247210395) [](ancestors = 247210395)